src/index.html

```diff
@@ -1,17 +1,123 @@
-<!doctype html>
-<html lang="en">
-  <head>
-    <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>HTML Form</title>
-    <link rel="stylesheet" href="./style.css">
-  </head>
-  <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
-  </body>
-</html>
+<!doctype html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport"
+    content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>HTML Form-01</title>
+  <link rel="stylesheet" href="./style.css">
+</head>
+
+<body>
+  <form action="" method="POST" autocomplete="off" onsubmit="onsubmit()">
+    <h1>HTML Form</h1>
+    <script type="text/javascript" src="./main.js"></script>
+
+    <fieldset class="block-information">
+      <legend>Personal information</legend>
+      <label for="" class="form-field" id="surname">
+        Surname:
+        <input type="text" name="surname" required>
+      </label>
+
+      <label for="" class="form-field" id="name">
+        Name:
+        <input type="text" name="name" required>
+      </label>
+
+      <label for="" class="form-field" id="age">
+        How old are you?
+        <input type="number" name="age" value="12" min="12" max="100" required>
+      </label>
+
+      <label for="" class="form-field" id="date-of-birth">
+        Full date of birth:
+        <input type="date" name="date-of-birth">
+      </label>
+
+      <label for="" class="form-field" id="terms">
+        I accept the term of the agreement
+        <input type="checkbox" name="terms">
+      </label>
+    </fieldset>
+
+    <fieldset class="block-information">
+      <legend>Registration</legend>
+      <label for="" class="form-field" id="email">
+        E-mail:
+        <input type="email" name="email" placeholder="email@example.com">
+      </label>
+      <label for="" class="form-field" id="password">
+        Password:
+        <input type="password" name="password" minlength="5" maxlength="15">
+      </label>
+    </fieldset>
+
+    <fieldset class="block-information">
+      <legend>An interesting fact about you!</legend>
+
+      <div class="form-field">
+        Do you have a cats?
+        <label for="" id="do-you-have-a-cats-yes">
+          <input type="radio" name="do-you-have-a-cats" value="yes">

```

Comment:
```suggestion
          <input type="radio" name="cats" value="yes">
```
просто cats будет достаточно